taxonomies/categories.txt

```diff
@@ -25234,6 +25279,13 @@ nl:Munster
 wikidata:en:Q765540
 oqali_family:en: fr:Fromages - Munster
 
+<en:Cheeses
+en:Saint-Paulin cheese from cow's milk (semi-hard cheese)
```

Comment:
This will make "semi-hard cheese" a synonym of "Saint-Paulin cheese from cow's milk".
+ if Saint-Paulin is always from cow's milk, then we should remove that precision in the name